Makefile

```diff
@@ -0,0 +1,8 @@
+.PHONY: build docker run
+
+
+docker:
+	docker build \
+	 --no-cache=true \
+		-t my-mongo \
```

Comment:
`my-mongo`?

Reply:
This is the name in the docker-compose file while doing testing (this can be seen in the dockercompose file in the security-secret-store) as the image needs to be built locally. We will need to adjust the name based on the decision if we are going to overwrite the original mongoDB image or use separate image/mongo service.